src/qml/qgismobileapp.qml

```diff
@@ -1000,6 +1021,24 @@ ApplicationWindow {
     toast.show(message)
   }
 
+  /** check if feature currently in trecking session **/
+  function featureOnTrack( layer, featureId ){
```

Comment:
Do you see any possibility to move that to the backend?
Keep a list of write protected layer, featureid, reason somewhere in the cpp code? I don't know where it would suit best, having some GUI items (like the FeatureListForm) depend on methods defined in the toplevel mobileapp qml sounds like a nightmare for reusing these modules in customized apps.